rotkehlchen/db/filtering.py

```diff
@@ -911,11 +912,13 @@ def make(
                     ),
                 )
         if entry_types is not None:
+            values = entry_types.values
+            behaviour = entry_types.behaviour
             filters.append(DBMultiIntegerFilter(
                 and_op=True,
                 column='entry_type',
-                values=[x.value for x in entry_types],
-                operator='IN',
+                values=[x.value for x in values],
+                operator=behaviour,
```

Comment:
Do not create temporary variable for no reason

```suggestion
                values=[x.value for x in entry_type.values],
                operator=entry_types.behaviour,
```